src/env.mjs

```diff
@@ -54,7 +54,10 @@ export const env = createEnv({
     MONGO_URL: process.env.MONGO_URL,
     NODE_ENV: process.env.NODE_ENV,
     NEXTAUTH_SECRET: process.env.NEXTAUTH_SECRET,
-    NEXT_PUBLIC_WS_PORT: process.env.NODE_ENV === "production" ? process.env.NEXT_PUBLIC_WS_PORT : "3001",
+    NEXT_PUBLIC_WS_PORT:
```

Comment:
Is this a formatter change?

Reply:
this one is automatic by my settings, I can just change it back